build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\Configuration.props" />
-  <Import Project="mono-runtimes.props" />
-  <Import Project="mono-runtimes.projitems" />
   <PropertyGroup>
-    <_MonoPath>..\..\external\mono</_MonoPath>
+    <_SourceTopDir>..\..</_SourceTopDir>
+    <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MonoPath>$(_SourceTopDir)\external\mono</_MonoPath>
```

Comment:
`$(_SourceTopDir)` - I don't like hard-coded constants all over the "code", makes it error prone should the value had to be changed. Plus, it makes reading the code easier.

I thought about MonoPath, just didn't want to introduce it in this diff as it is unrelated. I will open a separate PR for this.